operator/templates/tls-store-credentials.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $.Name}}-tls-store-credentials
+type: Opaque
+data:
+  keystore_password: Y2Fzc2FuZHJh
+  truststore_password: Y2Fzc2FuZHJh
```

Comment:
So, do we plan to make this some random string, or can this secret be configured by the user, or is this expected to be static?

Reply:
We could try to use sprigs uuid: 

{{ uuidv4 }}

Reply:
Random string would make sense. Sprig has all functions necessary for this. Though there could be implications regarding updates that we should look into first. To make it configurable by users not so much, because that's what's being removed by this PR.

Reply:
`{{ b64enc randAlphaNum 8 }}` could be used to randomize. Just needs to be Base64-encoded.

Reply:
Mmmmmm, but that would be regenerated every time the resource is deployed, right? That would be... not good :D

Reply:
Exactly, that's why I didn't look into this as part of this PR.